ps_checkout.php

```diff
@@ -1121,6 +1122,8 @@ public function hookActionFrontControllerSetMedia()
                 'checkout.payment.token.delete.modal.content' => $this->l('The following payment method will be deleted from your account:'),
                 'checkout.payment.token.delete.modal.confirm-button' => $this->l('Delete payment method'),
                 'checkout.payment.loader.processing-request' => $this->l('Please wait, we are processing your request'),
+                'APPLE_PAY_MERCHANT_SESSION_VALIDATION_ERROR' => $this->l('We’re unable to process your Apple Pay payment at the moment. This could be due to an issue verifying the payment setup for this website. Please try again later or choose a different payment method.'),
```

Comment:
The format doesn't really matter, since the translations are simple key-value pair. This is done because we get this error in Apple Pay and we can just use translation service to display the error as a user-friendly message.